code/src/components/AddTask.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { tasks } from 'reducers/tasks'
+import styled from 'styled-components'
+
+
+export const AddTaskForm = () => {
+  const [todo, setTodo] = useState('')
+  const dispatch = useDispatch()
+
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    dispatch(tasks.actions.addTask(todo))
+    setTodo('')
+  }
```

Comment:
to make this even better you could check so that the todo variable isn't empty before saving it as a todo? right now you could create empty todos.

```
const handleSubmit = (event) => {
  event.preventDefault()
  
  if (todo) {
    dispatch(todo.actions.addItem(todo))
    setTodo('')
  }
}
```